platform/core/Makefile

```diff
@@ -15,19 +15,19 @@ ios_targets := iPhone simulator
 tools_dir := tools
 android_core := PolyPodCoreAndroid/core/src/main/
 android_version := 24
-# Rust still uses old-ish r22 version of NDK.
-# For reference https://github.com/rust-lang/rust/pull/85806
-ndk_version := r22b
 google_repo := https://dl.google.com/android/repository
-ndk_lib := android-ndk-$(ndk_version)
-ndk=$(tools_dir)/NDK_$(ndk_version)
-ndk_archive = $(tools_dir)/NDK.zip
-export ANDROID_NDK_HOME := $(ndk)/$(ndk_lib)
 android_triples := aarch64-linux-android armv7-linux-androideabi x86_64-linux-android i686-linux-android
 android_binaries := $(patsubst %, target/%, $(android_triples))
 android_architectures := arm64-v8a armeabi-v7a x86_64 x86
 jni_libs_copy_path := $(android_core)/jniLibs
 android_core_libraries := $(patsubst %, $(jni_libs_copy_path)/%, $(android_architectures))
+# Android NDK fix
+# This is required due to Rust still using lgcc for NDK,
+# which was made osbsolete starting with NDK r23 in favor of lunwind.
+# The solution is to create a libgcc.a file in every clang architecture, which will link to -ilunwind.
+clang_path = ${ANDROID_NDK_HOME}/*/toolchains/llvm/prebuilt/*/lib64/clang/*/lib/*
+ndk_archs = aarch64 arm i386 x86_64
+libgcc_paths = $(patsubst %, $(clang_path)/%/libgcc.a, $(ndk_archs))
```

Comment:
This works in my local environment. In GH CI it does not work properly, as GH uses deprecated ndk-bundle, which is about to be altogether removed from GH env -> https://github.com/actions/virtual-environments/issues/5879

Reply:
I expect the error to be thrown due to the fact that directories do not exist, I am not sure what is the structure of the ndk-bundle.

Reply:
Ah, OK, I see what you mean. If the directory had existed, this would have mapped to a single, specific path, then the wildcards wouldn't actually show in the command line.

Reply:
Yes, if directories exist it maps to something like this:
```
/Users/ghenadiepusca/Library/Android/sdk/ndk/25.0.8775105/toolchains/llvm/prebuilt/darwin-x86_64/lib64/clang/14.0.6/lib/linux/aarch64/libgcc.a
```

Otherwise it throws the error if no directory path can be built, for example I did add `test` in the path, and this is the error:
```
/bin/sh: /Users/ghenadiepusca/Library/Android/sdk/ndk/*/test/toolchains/llvm/prebuilt/*/lib64/clang/*/lib/*/aarch64/libgcc.a: No such file or directory
```

Reply:
Thanks for your patience.